accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/installhook/AcToolInstallHook.java

```diff
@@ -52,9 +54,12 @@ public void execute(InstallContext context) throws PackageException {
     }
 
     private boolean shouldInstallInPhaseInstall(PackageProperties properties) {
-        return !Boolean.parseBoolean(properties.getProperty(PROPERTY_ACTOOL_INSTALL_AT_INSTALLED_PHASE));
```

Comment:
it needs to be inverse here (no `!`)

Reply:
right, good catch. We should rename the method to shouldInstallInPhaseInstalled at the same time, there is no phase INSTALL.

Reply:
Fixed in c6a9da28cf143ebbf43dc36e40d2056cf8efe13f